mikrotik/resource_dhcp_lease.go

```diff
@@ -33,9 +33,9 @@ func resourceLease() *schema.Resource {
 				Optional: true,
 			},
 			"blocked": &schema.Schema{
-				Type:     schema.TypeString,
+				Type:     schema.TypeBool,
 				Optional: true,
-				Default:  "false",
+				Default:  false,
```

Comment:
Why did this need to be changed? We probably need to test that upgrading from the old to the new terraform state works.

Reply:
Oops, my bad. I'll update this to keep the field the same, but a string -> bool conversion must happen since `BlockAccess` is defined as a bool on the `DhcpLease` struct.

Reply:
@ddelnano I've reverted the field type change + made other adjustments to ensure this works as expected.